src/httpServer/server.ts

```diff
@@ -278,7 +278,22 @@ export class DBOSHttpServer {
     const workflowResumeHandler = async (koaCtxt: Koa.Context) => {
       const workflowId = (koaCtxt.params as { workflow_id: string }).workflow_id;
       dbosExec.logger.info(`Resuming workflow with ID: ${workflowId}`);
-      await dbosExec.resumeWorkflow(workflowId);
+      try {
+        await dbosExec.resumeWorkflow(workflowId);
```

Comment:
We should return a 404 if the workflow is not found, not a 500

Reply:
But it's also an invalid operation, more than just "not found". I prefer to use 500 for all internal errors.